ddtrace/internal/wrapping/__init__.py

```diff
@@ -144,6 +144,8 @@ def wrap_bytecode(wrapper, wrapped):
             bc.Instr("RESUME", 0, lineno=lineno - 1),
             bc.Instr("PUSH_NULL", lineno=lineno),
         ]
+        if PY >= (3, 13):
+            instrs[1], instrs[2] = instrs[2], instrs[1]
```

Comment:
Why do we swap these for 3.13?

Reply:
I actually [searched](https://github.com/python/cpython/blob/f823910bbd4bf01ec3e1ab7b3cb1d77815138296/Python/flowgraph.c#L2655) for quite a while for the answer, and couldn't find it. @P403n1x87?